src/mono/sample/mbr/browser/runtime.js

```diff
@@ -2,17 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 var Module = { 
+    config: null,
+
+    // Called once the config file is loaded. The contents of the config file
+    // are passed as a JS object within the config parameter
+    onConfigLoaded: function (config) {
+        if (!config || config.error){
+            console.log("An error occured while loading the config file");
+            return;
+        }
+
+        Module.config = config;
```

Comment:
Is this is the function which could be replaced by user code, to receive post-load callback ?If so, do you expect the user code to modify `Module.config` directly ?

Reply:
Yes this function is a callback into user code. It gives the user the config so they can do whatever they want with it before sending it to the runtime. So this lets the user modify it at runtime as you had suggested earlier.